website/client/components/inventory/stable/index.vue

```diff
@@ -602,6 +605,19 @@
 
         this.searchTextThrottled = search;
       }, 250),
+      viewOptions: {
+        handler (newVal) {
+          if (!newVal) return;
+          if (!this.viewOptionsLoaded) return;
+          setLocalSetting(this.$route.name, JSON.stringify(newVal));
```

Comment:
this won't save the  `Hide Missing` filter

Reply:
all `hide ...` filters should be moved to `viewOptions` then those would be saved too